test/functorch/test_aotdispatch.py

```diff
@@ -2237,6 +2237,7 @@ def forward(self, x):
     xfail('cov'),
     xfail('chalf'),  # RuntimeError: "sum_cpu" not implemented for 'ComplexHalf'
     xfail('sparse.sampled_addmm'),
+    xfail('normal', 'number_mean'),  # TypeError: randn_like(): argument 'input' (position 1) must be Tensor, not float
```

Comment:
Doesn't this mean that we need to either wrap the mean into a tensor for this approach to work, or add support for numbers within `randn_like`?

Reply:
Thanks @lezcano, `'in_place'` `'normal'` opinfo (supported for `randn_like`) is testing this